modules/local/eggnog.nf

```diff
@@ -43,8 +51,7 @@ process EGGNOG_MAPPER {
         --no_file_comments \
         --cpu ${task.cpus} \
         --tax_scope 'prokaryota_broad' \
-        --dbmem \
```

Comment:
This flag makes by far the biggest difference in the annotation step. It shouldn't be removed.